staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -179,6 +195,8 @@ func newWatchCache(
 		keyFunc:             keyFunc,
 		getAttrsFunc:        getAttrsFunc,
 		cache:               make([]*watchCacheEvent, capacity),
+		lowerBoundCapacity:  min(capacity, defaultLowerBoundCapacity),
```

Comment:
FTR - I would like to get rid of passing "capacity" as an argument to newWatchCache at all (to get rid of all those magic constants). So it's fine for now, but you should add a TODO next to upperBoundCapacity and lowerBoundCapacity fields to get rid of them once we stop passing capacity as a parameter to watch cache.

Reply:
Done.